runtime/src/chpl-mem.c

```diff
@@ -93,15 +93,15 @@ int chpl_posix_memalign(void** ptr, size_t alignment, size_t size) {
 
 void* chpl_valloc(size_t size)
 {
-  return chpl_memalign(chpl_getHeapPageSize(), size);
+  return chpl_memalign(chpl_getSysPageSize(), size);
```

Comment:
Greg, note that I made this change (and it's not something panzone did). I'm having trouble coming up with a good defense for it. I was probably thinking we wouldn't want QIO to align its buffers to huge pages, but it's probably better to adjust the QIO calls. I think we should just remove that commit from this PR.